src/__tests__/Link.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import Enzyme, {mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import Link, {LinkColor} from '../Link';
+
+Enzyme.configure({adapter: new Adapter()});
```

Comment:
Is it possible to create some script for configuring Enzyme and run it before tests?